docs/local-setup.md

```diff
@@ -0,0 +1,27 @@
+### admission-alicloud
```

Comment:
```suggestion
# admission-alicloud
```

Also, the name of this file is too generic. Since it only covers the admission webhook, I think e.g. `local-setup-admission.md` would be a better name.

Reply:
This PR is following https://github.com/gardener/gardener-extension-provider-gcp/pull/112 and I do my best to make it as similar as possible and this file has very minimal changes - just the name of the cloud provider. Therefore I will keep it as it is now.